src/doc/trpl/compound-data-types.md

```diff
@@ -228,8 +228,8 @@ enum Character {
 }
 ```
 
-An `enum` variant can be defined as most normal types. Below are some example
-types have been listed which also would be allowed in an `enum`.
+Most normal types are allowed as the variant components of an `enum`. Below are
+some examples.
```

Comment:
Works for me, I'd agree "below are some examples" is still a little awkward, I was more focused on everything *else* about the original two sentences :smile:  I'm also open to suggestion on a more precise qualifier than "normal".